whylog/tests/utils.py

```diff
@@ -8,3 +11,33 @@ def get_path_to_config_files(cls, prefix_path):
         rules_path = os.path.join(prefix_path, 'rules.yaml')
         log_type_path = os.path.join(prefix_path, 'log_types.yaml')
         return [parsers_path, rules_path, log_type_path]
+
+
+class TestRemovingSettings(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        #This change was caused by fails os.names() for new directories.
+        #Renaming was necessary to avoid conflicts between test config directories
+        #and orginal independent non tests config directories
+        SettingsFactorySelector.WHYLOG_DIR = '.whylog_test'
+        cls.remove_settings_dirs()
+
+    @classmethod
+    def tearDownClass(cls):
+        #Removed test config directories if test failed
+        cls.remove_settings_dirs()
+
+    @classmethod
+    def remove_one(clas, path):
+        config_dir = os.path.join(path, SettingsFactorySelector.WHYLOG_DIR)
+        if os.path.isdir(config_dir):
+            shutil.rmtree(config_dir)
+
+    @classmethod
+    def remove_settings_dirs(cls):
+        path = os.getcwd()
```

Comment:
In that case find out who reviewed it and point it out to him. But fix the variable name as Ewa said. Just because some bad code is present in the repository does not mean that we should not fix it when we spot it.